overcloud-node-growvols.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Specify extra vars for extending logical volumes
+  import_playbook: /usr/share/ansible/tripleo-playbooks/cli-overcloud-node-growvols.yaml
+  vars:
+    growvols_args: >
```

Comment:
Will this set the same values for all roles?

Reply:
yes, at the moment we're not segregating based on role type as the bump up in space is not too large. If we segregate currently based on role it might increase complexity. We'll have to specify different playbooks for role type in that case.

Reply:
Okay, if there is no specific need for role based segragation, this should be fine.